go.mod

```diff
@@ -7,19 +7,19 @@ require (
 	github.com/go-logr/zapr v1.2.0
 	github.com/onsi/ginkgo/v2 v2.1.3
 	github.com/onsi/gomega v1.18.1
-	github.com/openshift/hypershift v0.0.0-20220511163932-ad143ad73524
-	github.com/openshift/library-go v0.0.0-20220112153822-ac82336bd076
-	github.com/spf13/cobra v1.2.1
+	github.com/openshift/hypershift v0.0.0-20220614212330-01bbfba6b681
```

Comment:
Please remember to update the version for downstream: https://gitlab.cee.redhat.com/mce-cpaas-midstream/hypershift/-/blob/multicluster-engine-2.0-rhel-8/upstream_sources.yaml#L53

Reply:
@rokej said he will help me with the DS update.